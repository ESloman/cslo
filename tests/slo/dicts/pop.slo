# slo: exp error

var map = {"a": 1, "b": 2, "c": 3};
println(map.pop("a"));     # 1
println(map);              # no 'a' key
println(map.pop("d", 4));  # 4
# this will error
println(map.pop("d"));     # nil
