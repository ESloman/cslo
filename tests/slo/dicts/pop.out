1
dict[3]: {c: 3, b: 2, }
nil
4
